spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/interpreter/ArrayConformanceSuite.scala

```diff
@@ -17,20 +17,18 @@ package za.co.absa.enceladus.conformance.interpreter
 
 import org.apache.spark.sql.functions._
 import org.mockito.Mockito.{mock, when => mockWhen}
+import org.scalatest.BeforeAndAfterAll
 import org.scalatest.funsuite.AnyFunSuite
-import org.scalatest.{BeforeAndAfterAll}
 import za.co.absa.enceladus.conformance.config.ConformanceConfig
 import za.co.absa.enceladus.conformance.datasource.DataSource
-import za.co.absa.enceladus.dao.MenasDAO
 import za.co.absa.enceladus.conformance.samples._
-import za.co.absa.enceladus.utils.fs.HdfsUtils
-import za.co.absa.enceladus.utils.testUtils.SparkTestBase
+import za.co.absa.enceladus.dao.MenasDAO
+import za.co.absa.enceladus.utils.testUtils.{HadoopFsTestBase, SparkTestBase}
 
-class ArrayConformanceSuite extends AnyFunSuite with SparkTestBase with BeforeAndAfterAll {
+class ArrayConformanceSuite extends AnyFunSuite with SparkTestBase with BeforeAndAfterAll with HadoopFsTestBase {
```

Comment:
What about having `EnceladusTestBase` as a combination of `SparkTestBase` and `HadoopFsTestBase` as they are combined so often?

Reply:
I have checked the numbers, `with SparkTestBase` is used about 64 times, and in conjunction of `with HadoopFsTestBase` it is used 16 times. This means, that I could only cover a minority of `SparkTestBase` encounters. I could change those 16 instances of `SparkTestBase with HadoopFsTestBase` for `EnceladusTestBase`.

I actually find this way much more explanatory. With this, you could directly see what is needed (spark, Hadoop FS), for other Enceladus-related suites, there may be other necessary traits, ... Or the other point - I don't think that all Enceladus test suites always need Spark + Hadoop FS.

So I am siding to keeping it as-is. But if you see where I am wrong, go ahead and point it out to me, please.